delfi/inference/SNPE.py

```diff
@@ -71,7 +74,7 @@ def loss(self, N, round_cl=1):
         N : int
             Number of training samples
         """
-        loss = -tt.mean(self.network.iws * self.network.lprobs)
+        loss = self.network.get_loss()
```

Comment:
I prefer how it was previously, without having a `get_loss` on the neural net class